scrapyrt/resources.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
+import demjson
```

Comment:
demjson does not belong to standard library https://www.python.org/dev/peps/pep-0008/#imports